gis/mod_tile/Portfile

```diff
@@ -5,12 +5,10 @@ PortGroup           active_variants 1.1
 PortGroup           boost 1.0
 PortGroup           github 1.0
 
-github.setup        openstreetmap mod_tile 0.5
+github.setup        openstreetmap mod_tile v0.7.2
```

Comment:
this doesn't look right, the "v" shouldn't be part of the "version". It should likely be:
".... mod_tile 0.72 v"

Reply:
I was assuming the version needs to match a [tag/release](https://github.com/openstreetmap/mod_tile/tags).  It's certainly some variation around `0.7.2` but that failed to fetch a release.